src/BankConsole/Decorators/BankStats.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using BankingSystem.Bank;
+
+namespace BankingSystem
+{
+    public class BankStats : Bank.Bank
+    {
+        private Bank.Bank bank;
+
+        public BankStats(Bank.Bank bank)
+        {
+            this.bank = bank;
+        }
+
+        // public override int NewAccount(AccountOrigin origin = AccountOrigin.LOCAL) {
```

Comment:
Either remove this code o fix it but avoid comment

Reply:
Ok, I fixed them, thank.